_pages/clients/minds.com.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Minds.com
+permalink: /minds.com/
+web:
+github: Minds/nostr-nips
+instances:
+- name: Minds.com
+  url: https://minds.com
+  comment: Minds is a social network with plans integrate a nostr relay and functions in their social network.  
+tags:
+- social
+- client
+platforms:
+- cross-platform
```

Comment:
please list platforms

Reply:
I'll create issues (hopefully it won't annoy the project maintainer) so I can identity.